django/db/backends/base/base.py

```diff
@@ -26,6 +27,8 @@
 NO_DB_ALIAS = "__no_db__"
 RAN_DB_VERSION_CHECK = set()
 
+logger = logging.getLogger("django.db.backends.base")
```

Comment:
```suggestion
logger = logging.getLogger("django.db.backends")
```
is probably sufficient 🤔

Reply:
I will make the change. I just used `django.db.backends.base` since the base.py file was in base module.

Reply:
I'm mostly mimicking existing behaviour. I can't find any logger with the `.base` module, but `django/db/backends/utils.py` has a plain `logger = logging.getLogger("django.db.backends")`. That's why I think we don't need the `base` :)

Reply:
`django.db.backends` logger is [documented](https://docs.djangoproject.com/en/4.1/ref/logging/#django-db-backends) as the SQL statements logger, so I will revert to `django.db.backends.base`.